### ISSUE TITLE
Compatibility with 5.5.0

### ISSUE THREAD

**Issue by Elepover** [Elepover/pmcenter#10]
On Telegram 5.5.0 and later, users can choose their messages to be forwarded without linking to them, instead, showing this JSON:

```
"forward_from_chat": {
      "id": -1001228946795,
      "title": "a user",
      "username": "HiddenSender",
      "type": "channel"
    },
    "forward_signature": "(user name)"
    ...
```

**Comment by Elepover** [comment]
Later we will add a new option to get pmcenter compatible with Telegram 5.5.0 and later.

`RecordMessageLinks`: Boolean. Record message ID links, is required when operating on messages that are anonymously forwarded to you.

```
"MessageLinks": {
    "OwnerChatID": (message ID in owner session),
    "UserID": {corresponding user's ID)
}

**Comment by Elepover** [comment]
After checking, the anti-flood system is not affected by this change.

**Comment by Elepover** [comment]
Fixed in commit [65da89d](https://github.com/Elepover/pmcenter/commit/65da89d7d16e3a657d0faf74622a1e409462fca7)
